libqtile/window.py

```diff
@@ -1387,9 +1400,9 @@ def _is_in_window(self, x, y, window):
         return (window.edges[0] <= x <= window.edges[2] and
                 window.edges[1] <= y <= window.edges[3])
 
-    def cmd_set_position(self, dx, dy):
+    def cmd_set_position(self, x, y):
         if self.floating:
-            self.tweak_float(dx, dy)
+            self.tweak_float(x, y)
```

Comment:
I'm cherry picking this into master, please rebase.

Reply:
Done.